apps/gp2-frontend/src/users/__tests__/export.test.ts

```diff
@@ -0,0 +1,233 @@
+import { gp2 as gp2Fixtures } from '@asap-hub/fixtures';
+import { gp2 as gp2Model } from '@asap-hub/model';
+
+import { Stringifier } from 'csv-stringify';
+
+import {
+  MAX_SQUIDEX_RESULTS,
+  squidexUsersResponseToStream,
+  userToCSV,
+} from '../export';
+
+afterEach(() => {
+  jest.clearAllMocks();
```

Comment:
resetAllMocks? But this spec is a copy/updated from CRN?

Reply:
yeah CRN is using the same